src/operator/subgraph/default_subgraph_op.h

```diff
@@ -35,18 +36,19 @@ namespace op {
  */
 class SubgraphSelector {
  public:
-  virtual ~SubgraphSelector() {
-  }
+  virtual ~SubgraphSelector() {}
   // Determine if the node should be selected for a subgraph.
-  virtual bool Select(const nnvm::Node &n) = 0;
+  virtual bool Select(const nnvm::Graph *g, const nnvm::Node &n) = 0;
   // Determine if the input node should be selected for a subgraph.
-  virtual bool SelectInput(const nnvm::Node &n, const nnvm::Node &new_node) = 0;
+  virtual bool SelectInput(const nnvm::Graph *g, const nnvm::Node &n,
+                           const nnvm::Node &new_node) = 0;
   // Determine if the output node should be selected for a subgraph.
-  virtual bool SelectOutput(const nnvm::Node &n, const nnvm::Node &new_node) = 0;
+  virtual bool SelectOutput(const nnvm::Graph *g, const nnvm::Node &n,
+                            const nnvm::Node &new_node) = 0;
   // Post processes pre-selected subgraph nodes. Return a list of nodes that
   // users want to keep in subgraph(s).
-  virtual std::vector<nnvm::Node*> Filter(nnvm::Graph* g,
-                                          const std::vector<nnvm::Node*>& candidates) {
+  virtual std::vector<nnvm::Node *> Filter(nnvm::Graph *g,
```

Comment:
No space between `Node` and `*`

Reply:
@reminisce clang-format puts a space between Node and *, if this is an issue i can manually modify it.